File: src/BuyAndHold.Api/Wallets/Domain/Wallet.cs
namespace BuyAndHold.Api.Wallets.Domain;
public record Wallet
{
    public long WalletId { get; set; }
    public string? Name { get; set; }
    public long UserId { get; set; }
    public bool IsDefault { get; set; }
    public ICollection<WalletSymbol>? Symbols { get; set; }
}

File: src/BuyAndHold.Client/src/app/wallets/domain/wallet.ts
import { WalletSymbol } from "./wallet-symbol";

export interface Wallet {
  walletId: number;
  name: string;
  userId: number;
  symbols?: WalletSymbol[] | null;
  isDefault: boolean;
  totalAmount?: number;
  expectedTotalAmount?: number;
}
